testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js

```diff
@@ -29,30 +29,30 @@ import 'generic_light.css!';
 QUnit.testStart(function() {
     const markup =
         '<div id="lookup"></div>\
-            <div id="secondLookup"></div>\
-            <div id="thirdLookup"></div>\
-            <div id="fourthLookup">\
-                <div data-options="dxTemplate: { name: \'test\' }">\
-                    <span data-bind="text: $data.id"></span>- <span data-bind="text: $data.caption"></span>\
-                </div>\
+        <div id="secondLookup"></div>\
```

Comment:
What about using backticks (`)?

Reply:
It's definitely good idea, but let's do it later, not in this cherry-pick

Reply:
ok